ember-container-query/src/components/container-query.ts

```diff
@@ -1,3 +1,5 @@
+import './container-query.css';
```

Comment:
It took me a few hours to realize that, in a v2 addon, I have to explicitly import stylesheets. Unfortunately, this hadn't been mentioned in either Embroider reference.

To help others realize this sooner, I have documented the need in [`ember-codemod-v1-to-v2`](https://github.com/ijlee2/ember-codemod-v1-to-v2).